src/Schema.js

```diff
@@ -409,7 +409,11 @@ class Schema {
 
             if (this.data[className][fieldName].startsWith('relation<')) {
               //For relations, drop the _Join table
-              return database.dropCollection(`_Join:${fieldName}:${className}`);
+              return database.collectionExists(`_Join:${fieldName}:${className}`).then(exist => {
```

Comment:
@Marco129 This is slightly racy. I don't expect people to be have many callers attempting to create and delete the same field many times, but I'd rather just avoid race conditions everywhere. Can you modify this to attempt the delete, and catch the error if the collection doesn't exist?